tests/unit/expected_test_results/parse_origins_from_text/simple-extraction-fr.json

```diff
@@ -0,0 +1,4 @@
+{
+   "lc" : "fr",
+   "origin_fr" : "Sucre France."
+}
```

Comment:
This is a bug, isn't it @stephanegigandet ?
We should have specific_ingredients